fineract-provider/src/main/java/org/apache/fineract/infrastructure/security/filter/TenantAwareTenantIdentifierFilter.java

```diff
@@ -95,7 +95,7 @@ public void doFilter(final ServletRequest req, final ServletResponse res, final
 
             // allows for Cross-Origin
             // Requests (CORs) to be performed against the platform API.
-            response.setHeader("Access-Control-Allow-Origin", "*");
+            response.setHeader("Access-Control-Allow-Origin", "https://mifos.org/mifos-x/");
```

Comment:
I'm not super familiar with CORS, but this looks curious, to me. What does this hard-coded URL mean and do exactly? We have a dedicated JIRA re. CORS (search). Perhaps it would be best to first and separately solve that, before adding secbugs?